vulnerabilityAlerts.json

```diff
@@ -0,0 +1,7 @@
+{
+    "vulnerabilityAlerts": {
+        "labels": [
+            "security"
+        ]
+    }
+}
```

Comment:
Same as above.

Reply:
Also added final line
[Add final line in json files](https://github.com/hatena/renovate-config/pull/81/commits/fe4814defd7901a522e88fe3ec1ab4bc856eccfa)